src/nnue/layers/sqr_clipped_relu.h

```diff
@@ -98,7 +98,7 @@ namespace Stockfish::Eval::NNUE::Layers {
         output[i] = static_cast<OutputType>(
             // really should be /127 but we need to make it fast
             // needs to be accounted for in the trainer
```

Comment:
looks like an unrelated code change

Reply:
I mean these are all technically unrelated.  The common thing is that they are all small non functional optimizations (most of them simplifications also) affecting general-64 and general-32.  This one combines two operations into one.  I'm not sure what's different about it in particular.

Reply:
it would definitely need to update the comment, and it increases at least the visual difference to the trainer code. Maybe @Sopel97 has an opinion on this.

Reply:
fine if the comment gets updated

Reply:
Sorry not updating the comment was definitely an oversight on my part.  Updated now.